web/themes/custom/usagov/templates/block--usagov-local-tasks.html.twig

```diff
@@ -0,0 +1,65 @@
+{# {% set site_url = url("<current>") %}
+{% if 'states' in site_url['#markup']|render|render %} #}
```

Comment:
While I'm going to suggest you go a different direction entirely, I wanted to point out that even this amount of digging around to find a value is a sign that we should look for another way -- either a pre-process hook, or, in the case of restricting text to a specific page, using one of Drupal's built-in mechanisms for that. 

Also, I don't think you meant to leave this commented out. :-)